packages/react-native-aztec/src/AztecView.js

```diff
@@ -234,6 +235,22 @@ class AztecView extends Component {
 		return focusedElement && focusedElement === this.aztecViewRef.current;
 	}
 
+	onRemoveMarkFormatting() {
+		UIManager.dispatchViewManagerCommand(
+			findNodeHandle( this.aztecViewRef.current ),
+			'onRemoveMarkFormatting',
+			[]
+		);
```

Comment:
I wonder if could use the function `dispatch` provided in this class.
```suggestion
		this.dispatch( AztecManager.Commands.onRemoveMarkFormatting );
```

Reply:
Initially, I tried that but had issues with the reference of the component. 

`dispatch` uses `this.aztecViewRef.current` as the `ReactTag` which I'm not sure if it's correct.

To be able to find the correct reference I needed to use `findNodeHandle`.

For example logging `this.aztecViewRef.current` returns:

```log
 {"_children": [], "_internalFiberInstanceHandleDEV": {"_debugHookTypes": null, "_debugNeedsRemount": false, "_debugOwner": {"_debugHookTypes": null, "_debugNeedsRemount": false, "_debugOwner": [FiberNode], "_debugSource": [Object], "actualDuration": 0.0012922286987304688, "actualStartTime": 2534308461.759458, "alternate": [FiberNode], "child": [FiberNode], "childLanes": 0, "deletions": null, "dependencies": null,
```

And `findNodeHandle( this.aztecViewRef.current )`:

```log
259
```